massgen/tests/test_cb_store_adversarial.py

```
@@ -0,0 +1,1062 @@
+        assert results.count(False) == 99
+        assert store.get_state("backend")["state"] == "open"
+
+    def test_concurrent_clear_and_increment(self) -> None:
+        store = InMemoryStore()
+        exceptions: list[BaseException] = []
+        exception_lock = threading.Lock()
+
+        def run_safely(action: Any) -> None:
+            try:
+                action("backend")
+            except BaseException as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `BaseException`

(BLE001)